libs/app-utils/main/src/lib/app-path/app-path.service.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import path from "path"
+
+export class AppPathService {
+  static join(...segments: string[]): string {
+    return path.join(...segments)
+  }
+}
```

Comment:
This is not needed; you can do `import path from "node:path"` on the renderer side.